apps/arc/arc.talon

```diff
@@ -0,0 +1,14 @@
+app: arc
+-
+tag(): browser
+tag(): user.tabs
+
+profile switch: user.arc_mod("shift-m")
```

Comment:
This doesn't work in Arc. There is the possibility of switching spaces with a shortcut, maybe you want to use that. Although the shortcuts for changing to a specific space (like ctrl-2) aren't working for me. Maybe something is interfering.

Reply:
You are right about the profile switch not working. For me the ctrl-<number> shortcuts are working. I'm not sure why they're not working for you. For now I'll just remove this command.